aspnetcore/signalr/cpp-client.md

```diff
@@ -0,0 +1,100 @@
+---
+title: ASP.NET Core SignalR C++ Client
+author: bradygaster
```

Comment:
everyone's putting my name in here. :) Feel free to use your own so you can get feedback on your docs. Or, if you want me to police them for you throw me under the bus. :)

Reply:
Think of it as me giving you credit for the C++ client 😛